### DISCUSSION TITLE
Changelog for Microsoft.CodeAnalysis.CSharp

### DISCUSSION THREAD

**Discussion by virzak** [dotnet/roslyn#72479] [Q&A]
Where can I see what's new in package Microsoft.CodeAnalysis.CSharp?

4.9.2 was released not long ago and I have no idea what's new.

**Comment by JoeRobich** [comment]
I don't know of a single place to see a high-level list of changes. If you are mostly interested in new features, those changes typically go into the first two preview releases. Roslyn releases do align with VS releases (4.x -> VS 17.x) and sometimes features will be written up in the new release blog posts.

PRs by milestone:
- [17.9 P1](https://github.com/dotnet/roslyn/milestone/157)
- [17.9 P2](https://github.com/dotnet/roslyn/milestone/159)
- [17.9 P3](https://github.com/dotnet/roslyn/milestone/162)
- [17.9](https://github.com/dotnet/roslyn/milestone/148)

[All commits between 4.8.0 and 4.9.2](https://github.com/dotnet/roslyn/compare/e091728607ca0fc9efca55ccfb3e59259c6b5a0a...9934fb9e3527e1c0c51314e57d4aab30f97e8f9e)
